### ISSUE TITLE
[docs] F.multilabel_soft_margin_loss does not list arg reduction, but lists outdated size_average

### ISSUE THREAD

**Issue by vadimkantorov** [pytorch/pytorch#67969]
https://pytorch.org/docs/stable/generated/torch.nn.functional.multilabel_soft_margin_loss.html?highlight=multilabel_soft_margin_loss#torch.nn.functional.multilabel_soft_margin_loss

Probably true for other functional loss doc pages

**Comment by jbschlosser** [comment]
Nice catch, and I second the idea of finally removing `size_average` / `reduce` from the docs.
